src/classes/FORM_ServiceGiftEntry.cls

```diff
@@ -174,18 +174,26 @@ public with sharing class FORM_ServiceGiftEntry {
     *
     * @param id: Form_Template__c record id
     *
-    * @return list: List of deleted Form_Template__c names
+    * @return FormTemplateWrapper: Wrapper object of the list of deleted template names and the result
+    * of the DML action
     */
     @AuraEnabled
-    public static String[] deleteFormTemplates(String[] ids) {
-        Form_Template__c[] templates = [SELECT Id, Name FROM Form_Template__c WHERE Id IN :ids];
-        delete templates;
-
+    public static String [] deleteFormTemplates(String[] ids) {
```

Comment:
With the change to use `{ deleteRecord } from lightning/uiRecordApi` I think this method can be removed altogether... ?  It doesn't seem to be in use anywhere else, unless I'm missing something. @psadie-sfdo can you verify?

Reply:
Yeah, it can be removed, but its called by the `FORM_Service` class, it seems this class was created to expose template action functionalities. But @psadie-sfdo can make that call.